### ISSUE TITLE
Broken with current version of Dash

### ISSUE THREAD

**Issue by sebastianludwig** [lurado/BetterAppleDocsets#6]
The structure of the dumped docset seems to have changed. Still investigating...

FYI @alex-swiftify

**Comment by alex-swiftify** [comment]
Thanks @sebastianludwig for your efforts!

Just a side question, is the extracted docset supposed to contain uncompressed Apple Help files in HTML format?
I’m looking for anway to generate an uncompressed version of Apple HTML help for our internal needs.

**Comment by sebastianludwig** [comment]
Yes, exactly this is how it used to work. Haven't figured out yet how it's working now or how to generate the HTML files. Will keep you posted.

**Comment by alex-swiftify** [comment]
@sebastianludwig, I have found that Dash no longer transfers the DocSet in HTML format since around Jan 2018 (but rather uncompresses the DocSet on the iOS device):
https://github.com/Kapeli/Dash-iOS/issues/66

I need uncompressed version(s) of Apple HTML Help files for our internal needs, and just got some help from Dash developer and a custom build of the "Apple Docs Helper" utility.
If you feel like you will benefit from that, I can share my findings with you - email me at alex(at)swiftify.com
